hms-app/src/components/HackathonDetails.tsx

```diff
@@ -46,8 +62,60 @@ export default function HackathonDetails(props: IProps) {
 
   useEffect(() => {
     loadSelectedHackathon()
+    setRelevantIdeaList([])
```

Comment:
I'm a little worried you built a bit of a race condition in here :thinking: I'm not sure if all these useEffect hooks execute at that same time or one after another... if it is the same time you call setRelevantIdeaList here and in the useEffect that is triggered by changes to ideaData...

Actually looking at this more you never seem to use ideaData by itself so you could remove it